sn_client/src/api.rs

```diff
@@ -384,6 +384,18 @@ impl Client {
             None
         };
 
+        // match self.network.put_record(record, record_to_verify).await {
+        //     Ok(spend) => Ok(spend),
+        //     Err(error) => match error {
+        //         // if record not matching what we expect, we should demove this CashNote from available_cash_notes
+        //         Error::Protocol(ProtocolError::RecordKindMismatch(RecordKind::Spend)) => {
+        //             warn!("RecordKind mismatch while trying to store a spend");
+        //             Err(Error::CouldNotVerifyTransfer(format!(
+        //                 "RecordKind mismatch while trying to store a spend"
+        //             )))
+        //         }
+        //     }
+        // }
```

Comment:
I don't think we check for record mismatch here? So the commented code can be removed I guess?